proto/qdrant/cloud/payment/v1/payment.proto

```diff
@@ -27,9 +27,37 @@ service PaymentService {
     // gRPC Gateway REST call
     option (google.api.http) = {get: "/api/payment/v1/accounts/{account_id}/payment-information/{payment_information_id}"};
   }
+  // Creates a new payment information for the account.
+  // This method is used to create a new payment information, which can then be connected to a payment provider (like Stripe).
+  // Required permissions:
+  // - write:payment_information
+  rpc CreatePaymentInformation(CreatePaymentInformationRequest) returns (CreatePaymentInformationResponse) {
+    // permissions
+    option (qdrant.cloud.common.v1.permissions) = "write:payment_information";
+    // custom account-id expression
+    option (qdrant.cloud.common.v1.account_id_expression) = "payment_information.account_id";
+    // gRPC Gateway REST call
+    option (google.api.http) = {post: "/api/payment/v1/accounts/{payment_information.account_id}/payment-information"};
+  }
+  // Updates the payment information for the account.
+  // This method is used to update the payment information details, such as billing address.
+  // Required permissions:
+  // - write:payment_information
+  rpc UpdatePaymentInformation(UpdatePaymentInformationRequest) returns (UpdatePaymentInformationResponse) {
+    // permissions
+    option (qdrant.cloud.common.v1.permissions) = "write:payment_information";
+    // custom account-id expression
+    option (qdrant.cloud.common.v1.account_id_expression) = "payment_information.account_id";
+    // gRPC Gateway REST call
+    option (google.api.http) = {
+      put: "/api/payment/v1/accounts/{payment_information.account_id}/payment-information/{payment_information.id}"
```

Comment:
When there are multiple resources, the path should be in plural, this should be `.../payment-informations/...`. That's another reason why I think naming this payment information isn't the best choice.